### PR TITLE
chore(cleanup storybook): cleanup storybook

### PR DESCRIPTION
**Pull request by rileylnapier**
## Description

- make sure storybook can access env vars from netlify
- make clientkey and auth optional and throw a warning
- fix some type errors storybook was causing

## Type of change

- [x] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> `Fix [#1]()`

### CODE REVIEW

File: packages/react-inbox/src/components/Messages2.0/Header.tsx [hunk 1]
```diff
@@ -322,12 +322,12 @@ const Header: React.FunctionComponent<IHeaderProps> = ({
...
       )}
       <div className="actions">
-        {messages.length > 0 && unreadMessageCount && (
+        {messages.length > 0 && unreadMessageCount ? (
           <MarkAllRead
             label={labels?.markAllAsRead}
             onClick={() => markAllAsRead()}
           />
-        )}
+        ) : null}
```
Comment (joshshowalter) [inline packages/react-inbox/src/components/Messages2.0/Header.tsx:330]: what is the need for this?

File: packages/react-provider/src/hooks/use-transport.ts [hunk 2]
```diff
@@ -55,11 +55,17 @@ const useTransport = ({
     }
 
     if (!clientKey && !authorization) {
-      throw new Error("Missing ClientKey or Authorization");
```
Comment (joshshowalter) [inline packages/react-provider/src/hooks/use-transport.ts:58]: we don't need to throw anymore?

File: packages/react-inbox/src/components/Messages2.0/Message.tsx [hunk 3]
```diff
@@ -44,7 +44,7 @@ const UnreadIndicator = styled.div<{ read?: IInboxMessagePreview["read"] }>(
 );
 
 const ClickableContainer = styled.a(({ theme }) => {
-  const primaryColor = theme.brand?.colors?.primary;
+  const primaryColor = theme.brand?.colors?.primary ?? "#9121c2";
```
Comment (joshshowalter) [inline packages/react-inbox/src/components/Messages2.0/Message.tsx:47]: what was the behavior before this?

File: packages/react-inbox/src/components/Messages2.0/Header.tsx (hunk 1 above)
Comment (rileylnapier) [reply packages/react-inbox/src/components/Messages2.0/Header.tsx:330]: it was rendering a 0 vs not rendering anything

File: packages/react-inbox/src/components/Messages2.0/Message.tsx (hunk 3 above)
Comment (rileylnapier) [reply packages/react-inbox/src/components/Messages2.0/Message.tsx:47]: if a brand can't be loaded it seems to render black so this is just a fallback

File: packages/react-provider/src/hooks/use-transport.ts (hunk 2 above)
Comment (rileylnapier) [reply packages/react-provider/src/hooks/use-transport.ts:58]: i got feedback that we shouldn't throw

File: packages/react-inbox/src/components/Messages2.0/Header.tsx (hunk 1 above)
Comment (joshshowalter) [reply packages/react-inbox/src/components/Messages2.0/Header.tsx:330]: that's v odd. `unreadMessageCount` being 0 should evaluate to falsy. Does the `MarkAllRead` component have some other value in it?

Comment (joshshowalter) [reply packages/react-inbox/src/components/Messages2.0/Header.tsx:330]: jk https://stackoverflow.com/questions/53048037/react-showing-0-instead-of-nothing-with-short-circuit-conditional-component

**Review by joshshowalter — APPROVED** [review]

**Pull request merged**